01_prepare_host.sh

```diff
@@ -60,15 +76,14 @@ if [ ! -f ${CENTOS_IMAGE} ] ; then
 fi
 popd
 
+sudo "${CONTAINER_RUNTIME}" wait -i 1000 ipa-downloader
```

Comment:
looks like this doesn't work when CONTAINER_RUNTIME is `docker`

https://jenkins.nordix.org/job/airship_metal3io_metal3_dev_env_integration_test_centos/13/console

Reply:
I did check to make sure docker had wait, but it doesn't support `-i`, I don't think we really need to specify the interval.